rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -115,26 +133,29 @@ def _decode_curve_remove_events(
         # 3. Withdrawal 2
         # etc.
         if return_event is None or len(withdrawal_events) == 0:
-            log.debug(
-                f'Expected to see a receive pool token event and deposit events for a curve pool, '
-                f'but have not found them. Tx_hash: {transaction.tx_hash.hex()} '
-                f'User address: {user_address}',
-            )
+            if maybe_user_address not in CURVE_DEPOSIT_CONTRACTS:
```

Comment:
Why this extra if? So if it's in curve deposits why it is fine to just `return None, []`?

There is no comment here or in the docstring of the function explaining the logic.
I guess it moves somewhere else? You need to write where/why